server/controllers/sign.js

```diff
@@ -39,6 +39,7 @@ exports.signup = async (request, response) => {
               email,
               school_id,
               birthdate,
+              percentage:15
```

Comment:
@MarwanElkhoudary It might be easier to change the percentage values later if you import this value from a variable in a new "percentage_constants.js" file instead of hard-coding it.
Organizationally, if you have all of the percentages for different user steps in one file, for example:
```
export const PERCENTAGE_AT_START = 15;
export const PERCENTAGE_AFTER_STUDIES = 30;
export const PERCENTAGE_AFTER_RESUME_UPLOAD = 45;
export const PERCENTAGE_FINISHED = 100;
```
Then it will be easy to find them and edit them later (for example, if you add another step to the signup process) since they're all together in one place.